tests/test_code_quality.py

```diff
@@ -93,9 +93,9 @@ def check_files(files, file_exclusions=None):
 
             test(fname)
 
-    check_files(["setup.py"])
+    check_files(["setup.py", "_setup_support.py"])
     check_tree('bin',          ['*'])
-    check_tree('bokeh',        ['*.py', '*.html', '*.js'], ["server/static"], ["__conda_version__.py"])
+    check_tree('bokeh',        ['*.py', '*.html', '*.js'], ["server/static"])
```

Comment:
Those changes won't be needed after PR #5751. We can resolve merge conflicts either way.